src/oc_stat_unit.erl

```diff
@@ -0,0 +1,59 @@
+-module(oc_stat_unit).
+
+-export([is_comparable/2,
+         must_convert/1,
+         convert/3]).
+
+%% ===================================================================
+%% API
+%% ===================================================================
+
+is_comparable(VUnit, MUnit) ->
+    {CVUnit, _} = canonical(VUnit),
+    {CMUnit, _} = canonical(MUnit),
+
+    case CMUnit of
+        arbitrary ->
+            false;
+        CVUnit ->
+            true;
+        _ ->
+            false
+    end.
+
+must_convert(native_time_unit) ->
+    true;
+must_convert(_) ->
+    false.
+
+convert(Value, From, To) ->
+    case is_comparable(From, To) of
+        false -> {error, {not_comparable_units, From, To}};
+        true ->
+            {_, FMult} = canonical(From),
+            {_, TMult} = canonical(To),
+            Value * FMult / TMult
```

Comment:
What happens when one of these is the atom `arbitrary`?

Reply:
in this case overriding unit in the view isn't allowed and it will stay `undefined`. No conversion will be performed - https://github.com/census-instrumentation/opencensus-erlang/pull/91/files/4baa2ba8ceb38403cb710dcf6a95ab8705c19666#diff-30ffb46b9429900cb285f935cef0ae5dR55